rust-src/concordium_base/src/id/account_holder.rs

```diff
@@ -704,11 +701,7 @@ where
 }
 
 #[allow(clippy::too_many_arguments)]
-pub fn create_unsigned_credential<
-    P: Pairing,
-    C: Curve<Scalar = P::ScalarField>,
-    AttributeType: Attribute<C::Scalar>,
->(
+pub fn create_unsigned_credential<P: Pairing, C: Curve<Scalar = P::ScalarField>, AttributeType>(
```

Comment:
**Non-blocking**: Is this clippy? I'm wondering why the trait bound declarations are not consistent 🤷

Reply:
Clippy doesn't like having bounds in both the parameter and the where clause, and suggested having them in the where clause. But I think it's probably better to have them in the parameter.